src/Kernel.package/CompiledMethod.class/instance/isDeprecated.st

```diff
@@ -1,9 +1,13 @@
 testing
 isDeprecated
-
-	^ self sendsAnySelectorOf: #(
-		#deprecated: 
-		#deprecated:on:in:
-		#deprecated:on:in:transformWith:
-		#deprecated:transformWith: 
-	)
\ No newline at end of file
+	"Object selectorsInProtocol: #deprecation"
+	(self
+		sendsAnySelectorOf:
+			#(#deprecated: #deprecated:on:in: #deprecated:on:in:transformWith: #deprecated:transformWith:))
+		ifTrue: [ ^ true ].
+	$-
+		split: self protocol asString
+		do: [ :each | 
+			each withBlanksCondensed = 'deprecated'
```

Comment:
Maybe we should add #asLowercase 

Also, maybe we should use #includesSubstring: instead of splitting, because if a method is in the `*Deprecated70` protocol it is deprecated.